docker/irods_client/tests/test_session.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from ibridges import Session
+
+
+def test_session(session, config, irods_env):
+    assert session.has_valid_irods_session()
+    assert ".".join(str(x) for x in session.server_version) == config["server_version"]
+    assert session.home == irods_env["irods_home"]
+    assert session.default_resc == irods_env["irods_default_resource"]
+    assert session.host == irods_env["irods_host"]
+    assert session.port == irods_env["irods_port"]
+    assert session.username == irods_env["irods_user_name"]
+    assert session.zone == irods_env["irods_zone_name"]
+
+
+def test_pam_password(session, config, irods_env):
+    if not config["can_write_pam_pass"]:
+        pytest.xfail("This iRods server cannot write pam passwords.")
```

Comment:
I think this test should also work. It might have something to do with the write-rights in the docker image. I will make a note and take a look. For this PR we cna leave it like this.

Reply:
I don't think it's a simple write issue though, since I'm also writing quite a few files, without problems. Maybe it's just in the root folder, or the directory is not made?